PocketKit/Sources/PocketKit/PocketSceneDelegate.swift

```diff
@@ -57,6 +57,9 @@ public class PocketSceneDelegate: UIResponder, UIWindowSceneDelegate {
                         },
                         premiumUpgradeViewModelFactory: { tracker, source in
```

Comment:
There is no strong reason. The reasoning behind this approach is to inject objects from the minimum upper scope where those objects are available. Because `AccountViewModel` has a `tracker` we use that one instead of picking it up directly from `Services`. It kinda feels like there's a bit more separation, but it's really not much of a difference

Reply:
but now that I'm looking at it, I'm questioning if I should instead change it and inject it directly  from `Services` so that the factory does not need to reason about it 😄

Reply:
changed it!